cmd/mesh/root.go

```diff
@@ -60,26 +59,15 @@ func GetRootCmd(args []string) *cobra.Command {
 	rootCmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
 
 	rootArgs := &rootArgs{}
-	diffArgs := &manDiffArgs{}
-	dumpArgs := &dumpArgs{}
 
-	ic := installCmd(rootArgs)
 	mc := manifestCmd(rootArgs)
-	mdc := manifestDiffCmd(rootArgs, diffArgs)
-	dpc := dumpProfileDefaultsCmd(rootArgs, dumpArgs)
+	pc := profileCmd(rootArgs)
```

Comment:
1) `maniestCmd` and `profileCmd` need to be exported so we can integrate them into istioctl

2) Can you move `addFlags(mc, rootArgs)` into manifestCmd so that the caller (e.g. istioctl) doesn't need to deal with it? And a similar change for the profile flags.

Reply:
Done.